### DISCUSSION TITLE
Is it possible to use swiftUI?

### DISCUSSION THREAD

**Discussion by jj10133** [organicmaps/organicmaps#7958] [Q&A]
Hi everyone consider me very new to swift and I love writing UI in swiftUI. To be honest I would like to change the UI for app but don't know where to begin from? Could someone explain as a brief overview? Plus what I'm saying is this even possible?

**Comment by biodranik** [comment]
Our current UI works tightly with C++ core and often uses Objective C. 

What are the battery/performance implications of using Swift UI?

**Reply by jj10133** [reply]
Tbh super new to swift don't know much about the internals atm...
